### ISSUE TITLE
[Bug] Integrated Dynamics Squeezer does not squeeze 

### ISSUE THREAD

**Issue by Cooltank10** [AllTheMods/All-the-mods-7-Sky#478]
I was playing on my single-player world and using the squeezer to squeeze cactuses into green dye for Ex Machinis gold upgrades. It was working as intended until a seemingly random point, after which I couldn't get the squeezer to squeeze at all. Even if there was nothing in the squeezer, it wouldn't lower when I jumped on it. The squeezer was not next to any storage areas as I think they can deposit into those. This started when I reset a squeezer early with no items in it (with a lever). After that, I couldn't get any squeezer to work on that world at all. On other worlds they work just fine even after I successfully reset a squeezer with a lever with no items and the squeezer being partially closed. It might be caused by having the world open, but paused for a extended amount of time (~45 min-1 hour). Would like to continue playing on that world so any feedback would be great!
Playing with no additional mods/shaders other than the ones in the ATM7ToTheSky pack. Also I had zero lag affecting anything as well.

**Comment by gaminggirl** [comment]
I seem to have accidentally stumbled into the exact same bug but in another pack, did you ever find a solution?

**Comment by Cooltank10** [comment]
Yes, I think I either had to restart my computer or restart Minecraft. It's been a while since I had that bug, but it seemed to resolve itself / easily fix. Lmk if restarting doesn't work. :)

**Comment by AbaddonFmL** [comment]
Had the same problem and solved it by realizing that I had admor with -25% gravity. It worked when I removed it and stopped working when I had it on.

**Comment by TommasoMalara** [comment]
> Had the same problem and solved it by realizing that I had admor with -25% gravity. It worked when I removed it and stopped working when I had it on.

OMG I was going crazy, I'm playing in a new world so i still have apotheosis mod disabled, i had a random golden chestplate and i couldn´t see the enchants on it, right after i read this comment i removed my chestplate and the squeezer started to work... what where the chances to get as first drop an armor with -gravity... insane
